### PR TITLE
setup velum

### PR DESCRIPTION
**Pull request by MaximilianMeister**
Signed-off-by: Maximilian Meister <mmeister@suse.de>

### CODE REVIEW

File: spec/support/helpers.rb [hunk 1]
```diff
@@ -114,6 +114,18 @@ def register
     click_on "Create Admin"
   end
 
+  def configure
+    visit "/setup"
+    fill_in "settings_dashboard", with: `hostname -f`
+    fill_in "settings_company_name", with: "SUSE LLC"
+    fill_in "settings_company_unit", with: "QA"
+    fill_in "settings_email", with: "containers.suse.de"
```
Comment (ereslibre) [inline spec/support/helpers.rb]: Typo, `containers@suse.de`?

Comment (MaximilianMeister) [inline spec/support/helpers.rb]: thanks

File: spec/support/helpers.rb [hunk 2]
```diff
@@ -114,6 +114,18 @@ def register
     click_on "Create Admin"
   end
 
+  def configure
```
Comment (mssola) [inline spec/support/helpers.rb]: For the future, this method should be configurable with some cooked data. I'm thinking on the non-ASCII characters bug.

Comment (MaximilianMeister) [inline spec/support/helpers.rb]: good idea.

Comment (MaximilianMeister) [inline spec/support/helpers.rb]: we could also add a set of tests to velum about that

**Pull request merged**